src/database/DatabaseBackend.h

```diff
@@ -2,8 +2,30 @@
 #include "DBOperation.h"
 #include "config/ConfigVariable.h"
 
-extern ConfigGroup db_backend_config;
-extern ConfigVariable<std::string> db_backend_type;
+
+#if NEW_BACKEND_CONFIG
```

Comment:
I think it is good to keep compatibility with old config files, perhaps define by default and offer switch to turn off "new" features?